src/configs/price.ts

```diff
@@ -3,61 +3,62 @@ import { PriceConfig } from "../types";
 export const DefaultPriceConfig: PriceConfig = {
   baseMultiplier: 1000,
   timeLabels: [
+    {
+      name: "Time: <1 Hour",
+      weight: 0.125,
+      value: 3600,
+      target: "Price: 100+ USD",
+    },
     {
       name: "Time: <1 Day",
       weight: 1,
       value: 3600 * 24,
-      target: "Price: 100-1000 USD",
+      target: "Price: 100+ USD",
     },
```

Comment:
seems duplicated in target price `Price: 100+ USD`, why?

Reply:
My mistake I guess the other one should be $12.5+ lol

Reply:
by the way, do we need to make a 12.5 USD bounty?